packages/gateway/src/generated/openapi/errors/ObjectTypesNotSynced.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ObjectTypeApiName } from "../components/ObjectTypeApiName";
+
+/**
+ * One or more of the requested object types are not synced into the ontology. Please reach out to your Ontology
+ * Administrator to re-index the object type(s) in Ontology Management Application.
```

Comment:
in theory this should only happen the very first time the object type is not synced

```suggestion
 * Administrator to index the object type(s) in Ontology Management Application.
```

Reply:
This is just autogenerated from the upstream docs, we don't control it.

Reply:
i can clearly read the file path, my bad